.github/workflows/ruby-publish.yml

```
@@ -0,0 +1,22 @@
+name: Publish Ruby Package
+
+on:
+  release:
+    types: [created]
```

release の作成時に実行するように指定。
実行コマンド`bundle exec rake release`でタグの生成も行われるが手動で生成されていても問題はないので、他SDKに合わせてrelease作成をトリガーに。

---

ファイル名は他SDKに合わせてreleaseでも良いかもですね（些事なのでどちらでも）

---

https://base.slack.com/archives/C0A86Q083/p1664352410765859?thread_ts=1664349404.400959&cid=C0A86Q083
```suggestion
    types: [published]
```